stdlib/src/builtin/float_literal.mojo

```diff
@@ -250,18 +250,26 @@ struct FloatLiteral(
         if not self._is_normal():
             return self
 
+        alias one = __mlir_attr.`#kgen.int_literal<1> : !kgen.int_literal`
```

Comment:
Why not just do this?
```suggestion
        alias one: IntLiteral = 1
```

Reply:
It does not compile:
```
❯ ./stdlib/scripts/run-tests.sh                                                                                                                                                                    
Creating build directory for building the stdlib running the tests in.                                                                                                                             
Packaging up the Standard Library.                                                                                                                                                                 
Included from /home/msaelices/src/mojo/stdlib/src/builtin/__init__.mojo:1:                                                                                                                         
/home/msaelices/src/mojo/stdlib/src/builtin/float_literal.mojo:264:30: error: no matching function in initialization                                                                               
                result = Self(truncated + plus_one)                                                                                                                                                
                         ~~~~^~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                
Included from /home/msaelices/src/mojo/stdlib/src/builtin/__init__.mojo:1:                                                                                                                         
/home/msaelices/src/mojo/stdlib/src/builtin/float_literal.mojo:51:8: note: candidate not viable: argument #1 cannot be converted from 'Int' to '!kgen.float_literal'                               
    fn __init__(inout self, value: Self.fp_type):                                                                                                                                                  
       ^                                                                                                                                                                                           
Included from /home/msaelices/src/mojo/stdlib/src/builtin/__init__.mojo:1:                                                                                                                         
/home/msaelices/src/mojo/stdlib/src/builtin/float_literal.mojo:60:8: note: candidate not viable: argument #1 cannot be converted from 'Int' to 'IntLiteral'                                        
    fn __init__(inout self, value: IntLiteral):                                                                                                                                                    
       ^                                                                                                                                                                                           
Included from /home/msaelices/src/mojo/stdlib/src/builtin/__init__.mojo:1:                                                                                                                         
/home/msaelices/src/mojo/stdlib/src/builtin/simd.mojo:180:8: note: candidate not viable: expected at most 1 positional argument, got 2   
```

Reply:
Maybe there is a workaround. I'm still learning the MLIR stuff and Mojo.

Reply:
Okay, thanks for checking!